fnproject/fn
# Add support for counting kdumps in pure-runner.

- What I did
Added support for counting the number of kdumps that are on a runner at the time the hostagent is started.
- How I did it
This is accomplished by adding a field to the runner status call.  Pure runner consumers who want to set this value may do so by passing the value in at the time the pure runner is created.
- How to verify it
Retrieve the status message and check for the presence of the field. 
- One line description for the changelog
Add support for counting kdumps in runner status message.
